.github/workflows/build-package-docs.yaml

```diff
@@ -120,3 +124,76 @@ jobs:
         name: package-docs-build
         path: build-directory/docs/docsite/ansible-package-docs-html-*.tar.gz
         retention-days: 7
+
+  deploy-package-docs:
+    needs: build-package-docs
+    runs-on: ubuntu-latest
+    environment:  # TODO create environment and specify it here
+    env:
+      dest-repo: owner/repo-name  # TODO create repo and specify here
+      dest-branch: main  # TODO specify repo branch here
+      user-email: username@example.com  # TODO specify email for commit
```

Comment:
Why not just use the "GitHub Actions" app user?

Reply:
that's certainly easier. I wasn't sure if we needed an email associated with the GitHub app or not, but I suppose the GH app is for generating the token with elevated permissions while the email here is simply for the commit message.

I wonder if it's necessary to put this in an `env` now...